ChangeLog

```diff
@@ -80,6 +80,11 @@ Release date: 2019-09-24
 
   Close #2834
 
+* Don't emit ``line-too-long`` for multilines when a
```

Comment:
Mind moving this to the new `2.5` section?

Reply:
@PCManticore i'm still working on it and now facing a small problem. 
In `pylint` there are two different messages : `bad-inline-option` and `unrecognized-inline-option` and i can't figure out what are the exact differences. Moreover the two corresponding unit-tests are very similar to me (`bad_line_option.py` and `unrecognized_inline_option.py`).
It would be nice if you could explain me a little bit further. 
Thanks in advance.

Reply:
Sure thing. `bad-inline-option` is emitted when the pragma is malformed, e.g. `# pylint: disable no-member`. Notice that message does not have the `=` sign. `unrecognized-inline-option` is emitted when a parameter in a pragma message is unrecognized, e.g. `# pylint: set-config: ...`, where `set-config` is unknown. We could probably squash these messages together but right now they have separate but related purposes.